src/components/tutorials-sidebar/tutorials-sidebar.module.css

```diff
@@ -1,16 +1,3 @@
-.root {
-  display: flex;
-  flex-direction: column;
-  gap: 20px;
```

Comment:
💖  I dig that adding `margin-bottom` on `BackToLink` removed the need for this layout stuff 🙌 Nicely done!